manifests/c/CoocooFroggy/FutureRestore-GUI/1.90/CoocooFroggy.FutureRestore-GUI.yaml

```
@@ -0,0 +1,17 @@
+PackageIdentifier: "CoocooFroggy.FutureRestore-GUI"
+PackageVersion: "1.90"
+PackageLocale: "en-US"
+Publisher: "CoocooFroggy"
```

```suggestion
Publisher: "Unknown"
```
Publisher mismatch. You'll have to change this by recompiling your application with `CoocooFroggy` as your publisher.